packages/destination-actions/src/destinations/metronome/sendEvent/index.ts

```diff
@@ -0,0 +1,94 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+function serializeEvent(event: Payload) {
+  // convert the timestamp into rfc3999
+  // TODO: we may want to enforce/drop non scalar property values here.
+  return {
+    ...event,
+    timestamp: new Date(event.timestamp).toISOString(),
+  }
+}
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Send Event',
+  description: 'Send an event to Metronome',
+  fields: {
+    transaction_id: {
+      type: 'string',
+      label: 'transaction_id',
+      // TODO: copy
+      description: 'The Metronome transaction id uniquely indentifies an event to ensure Metronome only processes each event once.',
```

Comment:
Capitalizing ID and fixing a typo:

```suggestion
      description: 'The Metronome transaction ID uniquely identifies an event to ensure Metronome only processes each event once.',
```